Clarify settings in default SSL/TLS (#41779)
elastic/elasticsearch: docs/reference/settings/security-settings.asciidoc

[[ssl-tls-settings]]
==== Default values for TLS/SSL settings
In general, the values below represent the default values for the various TLS
settings.

> These settings will always be prefixed based on the context in which they are

Reviewer: Not a big deal, but maybe consider:
```suggestion
The prefixes for these settings are based on the context in which they are
```
